### ISSUE TITLE
Annotation backup.velero.io/backup-volumes-excludes= does not lead to exclusion from backup

### ISSUE THREAD

**Issue by pitrh** [vmware-tanzu/velero#5762]
I followed the procedure from https://velero.io/docs/v1.10/file-system-backup/?#using-the-opt-out-approach to annotate volumes that should be excluded from file system backups (since they store databases that are backed up using database tools). However, the volumes are still backed up.

It looks like the annotations method was not in fact implemented.

Velero version 1.10.0.

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by qiuming-best** [comment]
@pitrh, maybe you need to specify `snapshot-volumes` value to `false` in the backup command like below 
`velero backup create t2 --include-namespaces nginx-example --default-volumes-to-fs-backup --snapshot-volumes=false`
or it will still backup the volume by taking a snapshot of it.

**Comment by pitrh** [comment]
> @pitrh, maybe you need to specify `snapshot-volumes` value to `false` in the backup command like below `velero backup create t2 --include-namespaces nginx-example --default-volumes-to-fs-backup --snapshot-volumes=false` or it will still backup the volume by taking a snapshot of it.

After a bit of futzing around we found that the method that *does* work is the one documented at https://velero.io/docs/v1.10/backup-reference/#exclude-specific-items-from-backup -- labeling the pvc and the pv with "velero.io/exclude-from-backup=true". Then the volume is actually excluded.

I would prefer the annotation method be actually implemented, but if that is for some reason not feasible, please correct the documentation so it at least stops being misleading.

**Comment by qiuming-best** [comment]
@pitrh Indeed, there are lots of rules to filter resources. and it's easy to be misleading.

the Opt-out approach will skip file system-level backup (Restic or Koipa) but it does not skip to taking snapshot (plugin or csi) for the same volume. you need to specify `snapshot-volumes` to false.

`velero.io/exclude-from-backup=true` label both works for  system-level backup or volume snapshot
